lib/tasks/refresh_framework_responses.rake

```diff
@@ -3,4 +3,13 @@ namespace :framework_responses do
   task refresh_data: :environment do
     FrameworkResponse.update_all("assessmentable_type = 'PersonEscortRecord', assessmentable_id = person_escort_record_id")
   end
+
+  desc 'Populate value_type on framework_responses'
+  task populate_value_type: :environment do
+    # Updates the response value type in batches of 1000
+    FrameworkResponse.where(value_type: nil).includes(:framework_question).in_batches.each do |response_batch|
```

Comment:
The bulk import does not trigger any callbacks, so no version or papertrail updated will occur. To ensure `updated_at` timestamp isn't changed I added the following option to the bulk import to avoid that: 
```
timestamps: false
```
https://github.com/zdennis/activerecord-import#activerecord-timestamps for reference

Reply:
Awesome stuff 👍